### DISCUSSION TITLE
Có nên giữ nguyên cách viết tắt của Quadratic Programming - QP?

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#64] [Q&A] [answered]
Mình đang revise phần SVM thì thấy từ này được viết tắt thành QP khá nhiều. 
Mình thấy nên dịch hẳn ra là `quy hoạch toàn phương` cho rõ ràng, thay vì `bài toán QP`.

Mọi người nghĩ sao nhỉ? @mlbvn/handson-ml2

**Comment by tiepvupsu** [comment]
Để QP cũng ok mà, viết tắt của Quy và Phương :D.

**Reply by ngcthuong** [reply]
Bên ngành mình QP là quantization parameter nha 😅

**Comment by minhduc0711** [comment, marked as answer]
Chốt là viết thành `quy hoạch toàn phương` nhé mọi người.
